### PR TITLE
WIP - Pulldown tweaks

### PR DESCRIPTION
**Pull request by squirrelo**
This changes a few more things found in the pulldown:
- Change true to Yes for static bool values
- Convert camel case to snake case for third party survey headers

### CODE REVIEW

File: knimin/lib/string_converter.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        dashed_case_str = self._all_cap_re.sub(r'\1-\2', sub2).lower()
+        return dashed_case_str.replace('--', '-')
+
+    def snakeToCamel(self, snake_cased_str):
+        return self._convertToCamel(snake_cased_str, "_")
+
+    def dashToCamel(self, snake_cased_str):
+        return self._convertToCamel(snake_cased_str, "-")
+
+    def _convertToCamel(self, snake_cased_str, separator):
```
Comment (mortonjt) [inline knimin/lib/string_converter.py]: I would be nice to have numpy-docs for all of these functions

**Comment by mortonjt** [comment]
One comment.  Otherwise :+1:

File: knimin/lib/string_converter.py (hunk 1 above)
Comment (squirrelo) [inline knimin/lib/string_converter.py]: I didn't write this, so thats honestly up to the developer of the library file.

Comment (mortonjt) [inline knimin/lib/string_converter.py]: Ah right.  Good catch.

**Comment by squirrelo** [comment]
Why did you merge a WIP pull request?

**Comment by mortonjt** [comment]
That was accident.  sry

**Comment by mortonjt** [comment]
I've got a revert PR in place

**Comment by squirrelo** [comment]
It's fine, it's all functionality we need, I was just going to continue adding the tweaks we needed onto this one .

**Pull request merged**